propagation/onboarding.md

```diff
@@ -0,0 +1,76 @@
+EO community long after your project ends.
+
+## Service Overview
+
+### Hosting Platform Onboarding Support
+
+APEx supports your project in selecting the appropriate hosting environment for your algorithm based on
+the [APEx Interoperability and Compliance Guidelines](../interoperability/algohostingenv.md). APEx also assists in the
+onboarding process, ensuring that your algorithm is available for execution on the selected platform and ready for
+further integration into the APEx Algorithm Catalogue.
+
+:::{.callout-tip}
+A full list of the known APEx-compliant algorithm hosting platforms is available [here](index.qmd#what-are-the-platforms-to-choose-from).
+:::
+
+### Automated Testing and Benchmarking
+
+To maintain the integrity of the algorithms, APEx conducts automated testing and benchmarking in collaboration with the
+respective project teams. This ensures that the algorithms remain functional and accessible over time. APEx offers this testing service for free, which reduces the maintenance burden on projects.
+
+To run the tests, APEx needs permission from the hosting platform, and an account with sufficient credits. Collaboration of the platform may be required to fulfill this requirement.
+
+### APEx Algorithm Catalogue
+
+The Algorithm Onboarding service supports result propagation through an algorithm catalogue, facilitating easy
+browsing and access to available algorithms. Users and other projects can also easily onboard their own algorithms,
+fostering collaboration and innovation within the EO community.
+
+The algorithm catalogue allows visitors to execute algorithms on the corresponding APEx-compliant hosting environment.
+For more details, please refer to [this section](#execute-a-service-from-the-apex-algorithm-catalogue).
+
+## Condiserations and Project Responsiblities
+
+### Algorithm Maintenance
+
+While APEx ensures the long-term availability of the algorithm within the catalogue, the creation, maintenance, and
+updating of the algorithm remain the responsibility of the original developer or the open-source community.
+
+### Algorithm Execution
+
+Algorithm execution is carried out one of the respective APEx-compliant algorithm hosting platform. By default, APEx
+does not provide direct access to processing resources on these platforms. However, projects can request processing
+resources through [ESA's Network of Resources (NoR)](https://portfolio.nor-discover.org/). When resources are requested
+for a specific algorithm, onboarded to the NoR, APEx can facilitate access to cloud processing resources on the
+corresponding platform as a broker.
+
+### Large-scale Data Processing
+
+For large-scale processing tasks, projects can request additional support through
+the [APEx upscaling service](./upscaling.md).
+
+## Onboard Your Algorithm on the APEx Algorithm Catalogue
+
+Stay tuned for more detailed instructions on how to onboard your algorithm into our catalogue.
+
+## Execute a service from the APEx Algorithm Catalogue
```

Comment:
I would remove this section for now 
I don't think that the Algorithm Catalogue should execute a service at least for now as this would have implications in the user account management that are not clearly identified

Reply:
To execute an algorithm, the user logs in to the platform, using the oidc standard, and then uses the relevant processing standard to start a job. The openEO web editor wizard is one example that already does this in practice, where there's one platform-agnostic UI.
If you are aware of specific issues in OGC API Processes, please let us know.